app.rb

```diff
@@ -0,0 +1,128 @@
+require_relative './book'
+require_relative './student'
+require_relative './classroom'
+require_relative './teacher'
+require_relative './rental'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def list_of_all_books
+    if @books.empty?
+      puts 'You have no recorded books yet!'
+    else
+      @books.each do |book|
+        puts "Title: \"#{book.title}\", Author: #{book.author}"
+      end
+      puts
+    end
+  end
+
+  def list_of_all_people
+    if @persons.empty?
+      puts 'You have no recorded people yet!'
+    else
+      @persons.each do |person|
+        category = person.is_a?(Student) ? '[Student]' : '[Teacher]'
+        puts "#{category} Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+      puts
+    end
+  end
+
+  def create_book
+    print 'Enter Book Title: '
+    title = gets.chomp
+    print 'Enter Book Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    # push new book to the end of the array
+    @books << book
+    puts 'Book is registered!'
+  end
+
+  def create_student
+    print 'Enter Age: '
+    age = gets.chomp
+    print 'Enter Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]:'
+    permission = gets.chomp
+    room = Classroom.new('A')
```

Comment:
- [ ] Please fix this syntax error Classroom suppose to be ClassRoom as in your class, please test the application  before submitting 
![image](https://user-images.githubusercontent.com/74744787/211108072-a86b98e6-6be5-4d8e-b154-550d1825a9a6.png)

Reply:
Okay, thanks a lot. I will walk through it